website/docs/r/orchestrated_virtual_machine_scale_set.html.markdown

```diff
@@ -101,6 +101,8 @@ resource "azurerm_orchestrated_virtual_machine_scale_set" "example" {
 
 * `termination_notification` - (Optional) A `termination_notification` block as defined below.
 
+* `user_data` - (Optional) The Base64-Encoded User Data which should be used for this Virtual Machine Scale Set.
```

Comment:
```suggestion
* `user_data_base64` - (Optional) The Base64-Encoded User Data which should be used for this Virtual Machine Scale Set.
```

Reply:
Have updated the property to `user_data_base64` as suggested, I originally mirrored the linux VMSS but this makes sense to me